lib/msal-browser/docs/cdn-usage.md

```diff
@@ -9,19 +9,23 @@ In addition to npm, `msal` can be consumed from Microsoft-hosted CDNs.
 
 ## Best Practices
 
-* Use the latest version of MSAL.js v2.
```

Comment:
Thanks for making this update, can you add a note though that says this documentation is for a legacy implementation and we instead recommend getting the latest MSAL.js and hosting it with your own application?

Reply:
Done